src/components/attendee/Toggler/Toggler.module.scss

```diff
@@ -33,35 +40,31 @@
   .slider:before {
     position: absolute;
     content: "";
-    height: 24px;
-    width: 24px;
-    left: 2px;
-    bottom: 2px;
-    background-color: rgba(255, 255, 255, 0.3);
+    height: space.empty(2);
+    width: space.empty(2);
+    bottom: space.gap(1);
+    left: space.gap(1);
+    background-color: theme.$c-glass-panel-background;
     -webkit-transition: 0.4s;
     transition: 0.4s;
   }
 
   input:checked + .slider:before {
-    background-color: rgba(255, 255, 255, 1);
+    background-color: theme.$c-panel-background;
   }
 
-  // input:focus + .slider {
-  //   box-shadow: 0 0 1px #2196f3;
-  // }
-
   input:checked + .slider:before {
     -webkit-transform: translateX(24px);
     -ms-transform: translateX(24px);
     transform: translateX(24px);
   }
 
   /* Rounded sliders */
-  .slider.round {
-    border-radius: 34px;
+  .sliderRound {
+    border-radius: space.empty(2.66);
```

Comment:
[NOTE] Oddly specific number. Maybe a comment would be useful of how exactly 2.66 is determined, maybe it might not even need a `space.empty()` but `space.toggler()` or something to make it understandable at first glance.

Reply:
replaced with a const (32px) [7f5a37b](https://github.com/sparkletown/sparkle/pull/2840/commits/7f5a37baa83617db540f69b073ae690c37f12ce6)